File: android/app/src/main/java/com/motiz88/rctmidi/webmidi/MIDIOutput.java
package com.motiz88.rctmidi.webmidi;
import com.motiz88.rctmidi.webmidi.errors.*;
import android.support.annotation.*;

public interface MIDIOutput extends MIDIPort {
  void send(@NonNull byte[] data, double timestamp) throws InvalidAccessError, InvalidStateError;
  void clear();
}

File: webmidi/MIDIOutput.js
import MIDIPort from './MIDIPort';
import MidiModule from '../MidiModule';

export default class MIDIOutput extends MIDIPort {
  send (data, timestamp) {
    return MidiModule.MIDIOutput_send(this.id, data, timestamp);
  }

  clear () {
    return MidiModule.MIDIOutput_clear(this.id);
  }
}
